app/updatemanager.go

```diff
@@ -281,6 +289,10 @@ func (m UpdateControlMap) Sanitize() {
 	}
 }
 
+func (u *UpdateControlMap) Expired() bool {
+	return time.Now().After(u.expiry)
```

Comment:
The ticket says to explicitly store the monotonic time, but the way I'm doing it now, I'm not really respecting it...
However, I see nothing wrong with this approach as is...

Reply:
I think you are actually using monotonic time.

From the [docs](https://golang.org/pkg/time/#hdr-Monotonic_Clocks):
"If Times t and u both contain monotonic clock readings, the operations t.After(u), t.Before(u), t.Equal(u), and t.Sub(u) are carried out using the monotonic clock readings", and in our case both of these are based on `time.Now()` which have monotonic clock reading.